internal/testhelpers/handler_mock.go

```diff
@@ -102,9 +103,11 @@ func MockHydrateCookieClient(t *testing.T, c *http.Client, u string) {
 	for _, c := range res.Cookies() {
 		if c.Name == config.DefaultSessionCookieName {
 			found = true
+			sessionCookie = c
```

Comment:
This will be overwritten by the loop reference, please use an explicit assignment